### ISSUE TITLE
On Blackberry Z10 BBOS 10.2.1.3062 - Open PDF into file - INVOCATION_BAD_REQUEST_ERROR

### ISSUE THREAD

**Issue by oolar** [blackberry/BB10-WebWorks-Samples#235]
I have tried the demo on Blackberry Z10 with BBOS 10.2.1.3062, version STL100-2 and I cannot open a pdf into Acrobat reader. It's always displaying  INVOCATION_BAD_REQUEST_ERROR.

**Comment by oolar** [comment]
I found the problem: I have a German Z10 and therefore path was not correcty configured into example. Anyway the error message is confusig.

**Comment by oolar** [comment]
Hello,
It was BB10-WebWorks-Samples-master\invoke\
I have found meanwhile the problem it was the path( I have a german Z10). Anyway the error code was confusing.

Issue was closed.

Freundliche Grüße / Best regards

Olimpiu Olar
App Development Manager

[cid:image003.png@01D0183C.57B9E790]

T +49 6571 9040 0 | F +49 6571 9040 40 | oolar@aeins.de | www.aeins.de
A EINS IT GmbH | Am kleinen Rotenberg 21 | 54516 Wittlich | Postfach 14 02 | 54504 Wittlich
Geschäftsführer: Andy Altmeyer | Handelsregister Nr.: HR B 12294 beim Amtsgericht Wittlich

Von: Junjie Huang [mailto:notifications@github.com]
Gesendet: Freitag, 12. Dezember 2014 19:41
An: blackberry/BB10-WebWorks-Samples
Cc: A EINS IT GmbH - Olimpiu Olar
Betreff: Re: [BB10-WebWorks-Samples] On Blackberry Z10 BBOS 10.2.1.3062 - Open PDF into file - INVOCATION_BAD_REQUEST_ERROR (#235)

Hi @oolarhttps://github.com/oolar, which demo?
